test/unittests/capi_test.cpp

```diff
@@ -145,6 +145,18 @@ TEST(capi, has_memory)
 
     fizzy_free_module(module_no_memory);
 
+    /* wat2wasm
+      (memory 0)
+    */
+    const auto wasm_memory_empty = from_hex("0061736d010000000503010000");
```

Comment:
Why is this duplicated in two test cases?

Reply:
One test here checks module inspection function `fizzy_module_has_memory`

The test below checks instance memory access functions.